x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/advanced_job_creator.ts

```diff
@@ -139,6 +145,9 @@ export class AdvancedJobCreator extends JobCreator {
     if (customRules !== null) {
       detector.custom_rules = customRules;
     }
+    if (useNull !== null) {
+      detector.use_null = useNull;
```

Comment:
If I set this to `false` in the JSON editor, and save the edit, on re-opening the JSON editor `"use_null": false` has been removed. I guess this is OK, as that is the default. Just wondering if that is the expected behaviour?